src/test/java/edu/hawaii/its/api/groupings/GroupingAddResultTest.java

```diff
@@ -27,7 +27,7 @@
 public class GroupingAddResultTest {
 
     @Value("${groupings.api.test.uh-usernames}")
-    private List<String> TEST_USERNAMES;
+    private List<String> TEST_UIDS;
```

Comment:
Final thing to change, please change TEST_UIDS back to TEST_USERNAMES for all test classes. This will be covered in another ticket, thanks!